crates/oxc_transformer/src/react_jsx/options.rs

```diff
@@ -1,14 +1,16 @@
-use serde::Deserialize;
-
 use oxc_semantic::Semantic;
+use serde::Deserialize;
 
-#[derive(Debug, Default, Clone, Copy, Deserialize)]
+#[derive(Debug, Default, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct ReactJsxOptions {
     /// Decides which runtime to use.
     pub runtime: ReactJsxRuntime,
     /// Toggles whether or not to throw an error if an XML namespaced tag name is used. e.g. `<f:image />`
     /// Though the JSX spec allows this, it is disabled by default since React's JSX does not currently have support for it.
     pub throw_if_namespace: Option<bool>,
+    /// Replaces the import source when importing functions. default to `react`
+    pub import_source: Option<String>,
```

Comment:
Can we try `Cow<'static, str>`, if the default is "react", we can manually implement `Default` for this struct.